src/components/IssueLabel.js

```diff
@@ -0,0 +1,54 @@
+import color from 'color-string'
+import { string } from 'prop-types'
+import styled from 'react-emotion'
+import {
+  colors,
+  fontSizes,
+  fontWeights,
+  lineHeights,
+  radii,
+  spacing,
+} from '../theme'
+import { cleanElement } from '../utils/utils'
+import { toAlpha } from '../utils/style'
+
+function getReadableColor(hex) {
+  // Color brightness is determined by the following formula:
+  // ((Red value X 299) + (Green value X 587) + (Blue value X 114)) / 1000
```

Comment:
I know you just copied the comment but maybe using a `*` is better than an `X` for this comment.